lms/envs/common.py

```diff
@@ -2323,6 +2323,7 @@
 }
 
 # E-Commerce API Configuration
+ECOMMERCE_EXTERNAL_URL_ROOT = None
```

Comment:
not a dealbreaker, but i suggest `ECOMMERCE_PUBLIC_URL` as a shorter and possibly more unambiguous name for this.